### PR TITLE
Make Microsoft.AspNet.Testing target generations

### PR DESCRIPTION
**Pull request by pranavkm**
cc @dougbu

### CODE REVIEW

File: src/Microsoft.AspNet.Testing/project.json [hunk 1]
```diff
@@ -7,10 +7,11 @@
     },
     "dependencies": {
         "Microsoft.Dnx.Runtime.Abstractions": "1.0.0-*",
-        "xunit.runner.aspnet": "2.0.0-aspnet-*"
```
Comment (pranavkm) [inline src/Microsoft.AspNet.Testing/project.json]: @dougbu https://github.com/aspnet/Testing/pull/76 doesn't tell why this was changed to point to the runner.

Comment (dougbu) [inline src/Microsoft.AspNet.Testing/project.json]: At one point it wouldn't compile w/o depending on the runner directly. If that's gone away, wunderbar.

**Comment by dougbu** [comment]
:shipit:

File: src/Microsoft.AspNet.Testing/project.json (hunk 1 above)
Comment (Eilon) [inline src/Microsoft.AspNet.Testing/project.json]: Yeah please kill it to death, if possible.

**Pull request merged**